packages/daycare/sources/core-tasks/section-execute-commit/task.py

```
@@ -116,7 +117,7 @@ def unique_lines(values: list[str]) -> list[str]:
 if default_branch is not None:
     branch_value = str(default_branch).strip()
 
-plan_text = read(path=plan_path_value)
+plan_text = read(path=plan_path_value)["content"]
```

Ruff (0.15.5)

[error] Undefined name `read`

(F821)